editor/editor_properties_array_dict.cpp

```diff
@@ -1216,3 +1216,225 @@ EditorPropertyDictionary::EditorPropertyDictionary() {
 	change_type->connect("id_pressed", callable_mp(this, &EditorPropertyDictionary::_change_type_menu));
 	changing_type_index = -1;
 }
+
+///////////////////// LOCALIZABLE STRING ///////////////////////////
+
+void EditorPropertyLocalizableString::_property_changed(const String &p_property, Variant p_value, const String &p_name, bool p_changing) {
+	if (p_property.begins_with("indices")) {
+		int index = p_property.get_slice("/", 1).to_int();
+		Dictionary dict = object->get_dict();
+		Variant key = dict.get_key_at_index(index);
+		dict[key] = p_value;
+
+		emit_changed(get_edited_property(), dict, "", true);
+
+		dict = dict.duplicate(); // Duplicate, so undo/redo works better\.
```

Comment:
```suggestion
		dict = dict.duplicate(); // Duplicate, so undo/redo works better.
```
Same below, and probably in the code you copy pasted originally.